tests/interface/testCmdTlmServer.cpp

```diff
@@ -18,6 +18,19 @@ typedef struct _serverData {
     struct sockaddr_in serverAddr;
 } serverData_t;
 
+// Data structure of the test telemetry, which has a bigger size
+typedef struct __attribute__((__packed__)) _telemetryTestBigStructure {
+    headerStructure_t header;
+    double dataA;
+    double dataB;
+} telemetryTestBigStructure_t;
+
+// Data structure of the test telemetry, which has a smaller size
+typedef struct __attribute__((__packed__)) _telemetryTestSmallStructure {
+    headerStructure_t header;
+    double data;
```

Comment:
Please consider making this something that is not a double and has a different length than a double (such as a char or short int -- or even a char array, though that would likely be larger than the other telemetry struct). That way if the code received data for the wrong struct, it will be much more obvious.

Reply:
Updated the code in commit: 7473889. Thanks!